microGallery_Android/app/src/main/kotlin/studio/lunabee/amicrogallery/bottomBar/BottomBarComposable.kt

```diff
@@ -0,0 +1,60 @@
+package studio.lunabee.amicrogallery.bottomBar
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.ExperimentalMaterial3ExpressiveApi
+import androidx.compose.material3.HorizontalFloatingToolbar
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import androidx.navigation.NavBackStackEntry
+import androidx.navigation.NavDestination.Companion.hasRoute
+import androidx.navigation.NavDestination.Companion.hierarchy
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.currentBackStackEntryAsState
+import studio.lunabee.amicrogallery.android.core.ui.component.topappbar.NavBarButton
+import studio.lunabee.amicrogallery.calendar.CalendarDestination
+import studio.lunabee.amicrogallery.core.ui.R
+import studio.lunabee.amicrogallery.lastmonth.LastMonthDestination
+import studio.lunabee.amicrogallery.untimed.UntimedDestination
+
+@OptIn(ExperimentalMaterial3ExpressiveApi::class)
+@Composable
+fun MicroGalleryBottomBar(
+    navController : NavHostController,
+    modifier: Modifier = Modifier,
+){
+    val currentBackStackEntry: NavBackStackEntry? by navController.currentBackStackEntryAsState()
+    Row(modifier = Modifier.fillMaxWidth().padding(15.dp), horizontalArrangement = Arrangement.Center) {
```

Comment:
You should add `15.dp` to `CoreSpacing` and use it from there. And it should be a multiple of `4` so `16` is better ([source](https://stackoverflow.com/a/69249994))